config/database.yml

```diff
@@ -7,7 +7,7 @@
 default: &default
   adapter: sqlite3
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
-  timeout: 5000
+  timeout: 15000
```

Comment:
Uploading 3 files at once can cause SQLite to timeout and result in a 500. If we wanna stick with SQLite, this is the best way to avoid that.